app/admin/__init__.py

```diff
@@ -0,0 +1,62 @@
+import warnings
+
+from flask import current_app, Flask, redirect, request, session, url_for
+from flask_admin import Admin, AdminIndexView, expose, helpers
+from flask_login import login_required, login_user, logout_user
+from http import HTTPStatus
+from urllib.parse import quote
+
+from .login import init_app as login_init_app, is_user_authenticated, LoginForm
+from .views import UsersView
+from ..oauth2 import get_service
+from ..models.db import db
+from ..models.orms.users import Users
+from ..utils.swagger_models import AuthModels
+
+
+class IndexView(AdminIndexView):
+    @expose('/')
+    def index(self):
+        if not is_user_authenticated():
+            return redirect(url_for('.login_view'))
+        return super(IndexView, self).index()
```

Comment:
Maybe make it like this
```py
        if is_user_authenticated():
            return super(IndexView, self).index()
        return redirect(url_for('.login_view'))
```
so it will be following the same format as following functions